installer/targets/cmdgen/install.bash

```diff
@@ -8,15 +8,30 @@ then
     echo "Installing dependency of GPSRCmdGen: mono-complete"
     sudo apt-get install mono-complete
 
-    tue-install-git http://github.com/kyordhel/GPSRCmdGen.git ~/src/GPSRCmdGen
+    tue-install-git https://github.com/tue-robotics/GPSRCmdGen.git ~/src/GPSRCmdGen
 
     echo "Making GPSRCmdGen"
     cd ~/src/GPSRCmdGen
     make
 
-    tue-install-info "GPSRCmdGen is only pulled and maked once. In the future you have to do this yourself.
-               cd ~/src/GPSRCmdGen
-               make"
-else
-    tue-install-debug "GPSRCmdGen already installed"
+else # GPSRCmdGen installed
```

Comment:
I think the `if-else` could be removed. First try to implement a normal flow for the tue fork with the use of tue-install-tue. With the git hash check in the end for the make. Then try to implement a temp fix for when you are not on the tue fork